README.md

```diff
@@ -13,7 +13,7 @@ A command-line interface to check the current statistics of COVID-19 of the worl
 
 ## Getting Started
 
-Getting started with the CoVID-19 CLI is very simple. Kindly follow the steps given below,
+Getting started with the CoVID-19 CLI is very simple. Kindly follow the steps given below.
 
 1.  Clone or download the repository using the command
```

Comment:
Seems like you haven't removed the extra whitespace between the point number and the first alphabetic character. Kindly complete this change at all the places based on our previous discussion.

Reply:
But this is the default behaviour of `Prettier`.
Should I still change this?

Reply:
Yes. It is recommended to remove the unnecessary things from the project files, which in this case is this extra whitespace. Having used "Prettier" in the past, I can recall that it does provide an option to change the configuration. However, for this minor change, you can do it manually and later work on your local configuration to avoid the extra whitespace in your future projects.